javascript/javascript1/week1/task1.js

```diff
@@ -0,0 +1,7 @@
+console.log("Age-ify!") // This is Age-ify 
+
+let yearOfBirth = 1994;
+let yearFuture = 2034;
+let age = yearFuture - yearOfBirth;
+
+console.log("You will be " + age + " years old in " + yearFuture)
```

Comment:
☺️You can also nest your variables inside your strings by using a template literal **${}** and wrapping the string up between inverted quotation marks: ``. It would look something like this: `This is a string with a nested ${variable} inside ` (Don't forget to add these quotation marks before and after the string: ` ` )